x-pack/plugins/observability_solution/observability/public/components/custom_threshold/helpers/threshold_unit.ts

```diff
@@ -8,13 +8,14 @@
 import { CustomThresholdExpressionMetric } from '../../../../common/custom_threshold_rule/types';
 import { decimalToPct, pctToDecimal } from './corrected_percent_convert';
 
-export const adjustThresholdBasedOnFormat = (
+export const convertToApiThreshold = (
```

Comment:
I renamed this function to indicate that for the percent unit, we will get `1` as the threshold from this function, which is the value that we will save in the rule's saved object.

In the ExpressionRow component, we will turn 1 to 100% only to show it nicely in the UI.